### ISSUE TITLE
[NSML] baseline 사용 문의

### ISSUE THREAD

**Issue by LVCSRer** [hunminjeongeum-korean-competition-2021/voice-recognition-performance-evaluation#26]
# 주제
- baseline 동작 여부 문의드립니다.

# 상황 (자세히 작성해주시면 좋습니다.)
- 많은 분들이 이미 좋은 결과를 제출하고 계신데 아직 기본적인 실행도 못해본 상황이네요. baseline 을 clone 하고 nsml run -d stt_1 -g 1 으로  실행하면 아래와 같이 로그가 나오고 가만히 있는데 이게 제대로 돌아가고 있는 것인지 문제가 생겨서 중단된 것인지 모르겠습니다.
- 17분 가량 지나서 nsml model ls  명령을 확인해도 생성된 모델은 없습니다.

NSML Log start!
=============
== PyTorch ==
=============

NVIDIA Release 21.07 (build 25165078)
PyTorch Version 1.10.0a0+ecc3718

Container image Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.

Copyright (c) 2014-2021 Facebook Inc.
Copyright (c) 2011-2014 Idiap Research Institute (Ronan Collobert)
Copyright (c) 2012-2014 Deepmind Technologies    (Koray Kavukcuoglu)
Copyright (c) 2011-2012 NEC Laboratories America (Koray Kavukcuoglu)
Copyright (c) 2011-2013 NYU                      (Clement Farabet)
Copyright (c) 2006-2010 NEC Laboratories America (Ronan Collobert, Leon Bottou, Iain Melvin, Jason Weston)
Copyright (c) 2006      Idiap Research Institute (Samy Bengio)
Copyright (c) 2001-2004 Idiap Research Institute (Ronan Collobert, Samy Bengio, Johnny Mariethoz)
Copyright (c) 2015      Google Inc.
Copyright (c) 2015      Yangqing Jia
Copyright (c) 2013-2016 The Caffe contributors
All rights reserved.

NVIDIA Deep Learning Profiler (dlprof) Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.

Various files include modifications (c) NVIDIA CORPORATION.  All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

NOTE: Legacy NVIDIA Driver detected.  Compatibility mode ENABLED.

NOTE: MOFED driver for multi-node communication was not detected.
      Multi-node communication performance may be reduced.

torch version:  1.10.0a0+ecc3718
100%|██████████| 97.8M/97.8M [00:00<00:00, 106MB/s]
/app/stt_model.py:16: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  seq = torch.tensor(torch.eq(seq, 0), dtype=torch.float32)
main.py:86: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  mask = torch.tensor(mask, dtype=loss_.dtype)
main.py:96: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  accuracies = torch.tensor(accuracies, dtype=torch.float32)
main.py:97: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  mask = torch.tensor(mask, dtype=torch.float32)

# 내용 (첨부 가능한 이미지 추가해주시면 더 좋습니다.)
- 

# 기타 문의 사항
-

**Comment by senti-mkyu** [comment]
저희도 baseline 코드 수정해서 진행하고있는데, 잘 작동합니다.
베이스라인 코드 그대로 사용시 매 에포크마다 결과를 출력하게 되어있는데, epoch for문 내에 log_interval로 몇 iteration마다 몇 loss를 기록하는지 print 해보심이 좋을 것 같습니다

**Comment by LVCSRer** [comment]
> 저희도 baseline 코드 수정해서 진행하고있는데, 잘 작동합니다. 베이스라인 코드 그대로 사용시 매 에포크마다 결과를 출력하게 되어있는데, epoch for문 내에 log_interval로 몇 iteration마다 몇 loss를 기록하는지 print 해보심이 좋을 것 같습니다

조언 주신대로 진행해보겠습니다. 도움 주셔서 감사드립니다.
